core/unit_test/TestUniqueToken.hpp

```diff
@@ -166,8 +166,11 @@ TEST(TEST_CATEGORY, unique_token_global) {
 }
 
 TEST(TEST_CATEGORY, unique_token_instance) {
+  // FIXME_OPENMPTARGET - Not yet implemented.
+#if !defined(KOKKOS_ENABLE_OPENMPTARGET)
```

Comment:
I thought this works? Its just the acquire_team_unique_token which fails?

Reply:
Yes right, enabled it back.